packages/react-native/scripts/xcode/ccache.conf

```diff
@@ -0,0 +1,5 @@
+# See https://ccache.dev/manual/4.3.html#_configuration_options for details and available options
+sloppiness = clang_index_store,file_stat_matches,include_file_ctime,include_file_mtime,ivfsoverlay,pch_defines,modules,system_headers,time_macros
+file_clone = true
+depend_mode = true
+inode_cache = true
```

Comment:
I've been contemplating adding the `compiler_check = content` mentioned in the guide: https://reactnative.dev/docs/build-speed#using-this-approach-on-a-ci .. I believe it'll make compilations marginally slower, but it would make this even more useful on CI.

Reply:
Have you checked how cacheable this config is? We've been using this on CI and have typically seen cache hits above 95% when bumping patch releases: https://github.com/microsoft/react-native-test-app/blob/449fe0254d9a8aebe89322fc182ddf7643e07c23/.ccache/ccache.conf

Reply:
This config is the one suggested on our "Speeding up your Build phase" guide, just converted from env variables to the ccache conf format. I've seen 100% hit rates on a newly instantiated template while deleting the `ios/build` directory in between builds.